website/html/views.py

```diff
@@ -522,7 +527,7 @@ def myvessels(request, get_form=False, action_summary="", action_detail="", remo
   
   # this user's number of donations, max vessels, total vessels and free credits
   my_donations = interface.get_donations(user)
-  my_max_vessels = interface.get_available_vessel_credits(user)	
+  my_max_vessels = interface.get_available_vessel_credits(user) 

```

Comment:
Addressed in 16955796fc708c38cc5b5fd5a8dc402a389405c3, but there's much more to do.